app/views/layouts/application.html.erb

```diff
@@ -1,27 +1,27 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Cansei</title>
+    <title><%= Settings.site_title %></title>
     <%= csrf_meta_tags %>
 
-    <%= stylesheet_link_tag    'application', media: 'all' %>
+    <%= stylesheet_link_tag 'application', media: 'all' %>
     <%= javascript_include_tag 'application' %>
   </head>
 
   <body>
     <nav>
       <% if current_user %>
         <p>
-          Logado como <%= current_user.username %>
+          Logado como @<%= current_user.username %>
           (<%= link_to 'Sair', logout_path, method: :delete %>)
         </p>
       <% else %>
         <p><%= link_to 'Entrar com sua conta do Twitter', '/auth/twitter' %></p>
       <% end %>
     </nav>
 
-    <h1>Como funciona</h1>
-    <p>Breve explicação de como funciona</p>
+    <h1><%= Settings.site_title %></h1>
+    <p><%= Settings.brief_explanation %></p>
```

Comment:
will definitely conflict with #17. let's merge this asap and i'll resolve conflicts.